src/Core/Algorithms/Base/AlgorithmBase.cc

```diff
@@ -203,11 +203,6 @@ Variable SCIRun::Core::Algorithms::makeVariable(const std::string& name, const V
   return Variable(Name(name), value);
 }
 
-Variable SCIRun::Core::Algorithms::makeVariable(const Name& name, const Variable::Value& value)
```

Comment:
Nice to delete this, but the references were not cleaned up--see broken build. Just replace with constructor call.